<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TBProject</class>
 <widget class="QWidget" name="TBProject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1196</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox
{
	border-width: 1px;
	border-style: solid;
	border-color:  gray;
	border-radius: 3px;
	margin-top: 1ex;
	padding: 10px;
}
QGroupBox::title
{
	subcontrol-origin: margin;
	subcontrol-position: top left;
	left: 10px;
	margin-left: 0px;
	padding: 0px 1px;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" rowspan="3">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Schema</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Layers</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_numLayer">
          <item>
           <property name="text">
            <string>3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="Schema3Layer" name="widget_schema" native="true">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>400</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">image: url(:/drawable/3L.tiff);</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Bias</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Min:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_bias_min">
        <property name="minimum">
         <double>-20.000000000000000</double>
        </property>
        <property name="maximum">
         <double>20.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Max:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_bias_max">
        <property name="minimum">
         <double>-20.000000000000000</double>
        </property>
        <property name="maximum">
         <double>20.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Num:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_bias_num"/>
      </item>
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>nProc</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_nproc">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>Parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="5" column="1">
       <widget class="QLineEdit" name="lineEdit_lead1_4"/>
      </item>
      <item row="10" column="1" colspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_coupling_1"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_coupling_2"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="lineEdit_lead1_5"/>
      </item>
      <item row="0" column="2">
       <widget class="QComboBox" name="comboBox_material_2"/>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Coupling</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QComboBox" name="comboBox_material_3"/>
      </item>
      <item row="2" column="3">
       <widget class="QLineEdit" name="lineEdit_lead2_1"/>
      </item>
      <item row="8" column="2">
       <widget class="QLineEdit" name="lineEdit_B1_7"/>
      </item>
      <item row="3" column="3">
       <widget class="QLineEdit" name="lineEdit_lead2_2"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_lead1_1"/>
      </item>
      <item row="7" column="2">
       <widget class="QLineEdit" name="lineEdit_B1_6"/>
      </item>
      <item row="6" column="2">
       <widget class="QLineEdit" name="lineEdit_B1_5"/>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="lineEdit_lead1_6"/>
      </item>
      <item row="1" column="1">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QLineEdit" name="lineEdit_lead2_3"/>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="lineEdit_B1_2"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>tâ†“</string>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QLineEdit" name="lineEdit_lead2_5"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Material</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Angle Î¸</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEdit_lead1_3"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_1">
        <property name="text">
         <string>Îµâ†‘</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_8">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Temperature</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="Line" name="line_1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QLineEdit" name="lineEdit_lead2_4"/>
      </item>
      <item row="8" column="3">
       <widget class="QLineEdit" name="lineEdit_lead2_7"/>
      </item>
      <item row="4" column="2">
       <widget class="QLineEdit" name="lineEdit_B1_3"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>tâ†‘</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLineEdit" name="lineEdit_B1_4"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Îµâ†“</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox_material_1"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Thickness</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="lineEdit_B1_1"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit_lead1_2"/>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="lineEdit_lead1_7"/>
      </item>
      <item row="7" column="3">
       <widget class="QLineEdit" name="lineEdit_lead2_6"/>
      </item>
      <item row="9" column="0" colspan="4">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="Line" name="line_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Plot</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton_plot_Tprl">
        <property name="text">
         <string>V - Tprl</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton_plot_Tnml">
        <property name="text">
         <string>V - Tnml</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButton_plot_Iz">
        <property name="text">
         <string>V - Iz</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton_plot_I">
        <property name="text">
         <string>V - I</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButton_plot_TMR">
        <property name="text">
         <string>V - TMR</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pushButton_5">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer_7">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Schema3Layer</class>
   <extends>QWidget</extends>
   <header>schema3layer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
